Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -290,6 +306,28 @@ private function makeArrayFromValue($name, $value): array
             return $value;
         }
 
-        throw new NostoException(sprintf('Can not get value for filter: %s', $name));
+        // @codingStandardsIgnoreStart
+        $this->logger->debugWithSource(
```

Comment:
These logging statements seem bloated. The logger erasure should be cleaned up. We have 5-7 lines for one log statement and this is drastically reducing the readability of the code.